references/obj_detection/utils.py

```diff
@@ -3,8 +3,39 @@
 # This program is licensed under the Apache License version 2.
 # See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
 
+from typing import Dict, List
+
+import cv2
 import matplotlib.pyplot as plt
 import numpy as np
+from matplotlib.cm import get_cmap
+
+
+def plot_samples(images, targets: List[Dict[str, np.ndarray]], classes: List[str]) -> None:
+    cmap = get_cmap('gist_rainbow', len(classes))
+    # Unnormalize image
+    nb_samples = min(len(images), 4)
+    _, axes = plt.subplots(1, nb_samples, figsize=(20, 5))
+    for idx in range(nb_samples):
+        img = (255 * images[idx].numpy()).round().clip(0, 255).astype(np.uint8)
+        if img.shape[0] == 3 and img.shape[2] != 3:
+            img = img.transpose(1, 2, 0)
+        target = img.copy()
+        for box, class_idx in zip(targets[idx]['boxes'].numpy(), targets[idx]['labels']):
+            r, g, b, _ = cmap(class_idx.numpy())
+            color = int(round(255 * r)), int(round(255 * g)), int(round(255 * b))
+            cv2.rectangle(target, (int(box[0]), int(box[1])), (int(box[2]), int(box[3])), color, 1)
```

Comment:
Could you increase the line width please? :pray:

Reply:
Thanks :) It is resolved